06-11-2018/Italent2BoardCreation/src/main/java/com/italent2/controller/MainClass.java

```diff
@@ -0,0 +1,29 @@
+package com.italent2.controller;
+
+import java.io.IOException;
+import java.text.ParseException;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpResponseException;
+
+import com.italent2.daoImpl.DatabaseConnectionDaoImpl;
+import com.italent2.model.Messages;
+import com.italent2.service.BoardCreationAndMessagePosting;
+
+public class MainClass 
+{
+	public static void main(String args[]) throws HttpResponseException, IOException, ParseException
```

Comment:
check exception handling. how to handle exception in main method check online for this. let me know if you need anything on this.